internal/db/merge.go

```diff
@@ -129,12 +245,14 @@ func (m *mergeQueue) done(docID string) {
 }
 
 type mergeProcessor struct {
-	txn        datastore.Txn
-	lsys       linking.LinkSystem
-	mCRDTs     map[string]merklecrdt.MerkleCRDT
-	col        *collection
-	dsKey      core.DataStoreKey
-	composites *list.List
+	txn                          datastore.Txn
+	lsys                         linking.LinkSystem
+	mCRDTs                       map[string]merklecrdt.MerkleCRDT
+	col                          *collection
+	dsKey                        core.DataStoreKey
+	blocks                       *list.List
+	pendingEncryptionKeyRequests map[core.EncStoreDocKey]struct{}
+	hasPendingCompositeBlock     bool
```

Comment:
suggestion: I know this is a private struct, but would be helpful to have a quick one-liner documentation on the last three.

Reply:
added some docs